components/CountryDetails/index.js

```diff
@@ -0,0 +1,67 @@
+import StyledFlag from "../StyledFlag";
+import StyledDiv from "../StyledDiv";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledLink = styled.a`
+  font-size: 2rem;
+  position: relative;
+  right: 9rem;
+  top: 1rem;
+`;
+
+export default function CountryDetails({ selectedCountry }) {
+  function addSeparators(number) {
+    return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
+  }
+  if (!selectedCountry) return <h2>is Loading</h2>;
```

Comment:
I am not sure but aren't the `{...}` missing for the function body? But might be some JSX functionality I have not heard about yet 🙂